src/main/java/org/jboss/tools/intellij/openshift/utils/odo/OdoCli.java

```diff
@@ -229,26 +230,22 @@ public void deleteApplication(String project, String application) throws IOExcep
 
     @Override
     public void push(String project, String application, String context, String component) throws IOException {
-        ExecHelper.executeWithTerminal(this.project, WINDOW_TITLE, new File(context), true, envVars, command, "push");
+        ExecHelper.executeWithTerminal(this.project, WINDOW_TITLE, new File(context), true, envVars, null, null, null, command, "push");
```

Comment:
@sbouchet, @jeffmaury, @lstocchi: please verify this fix for breaking changes in intellij-common

Reply:
This would work but it's ugly imo. It should be transparent for plugins using common if there is no need to setup those optional things. Maybe i deleted that method signature accidentally during those changes. I'll add it again. Would it be better, no?

Reply:
https://github.com/redhat-developer/intellij-common/pull/147 was merged, updated my PR